gadget/gadget.go

```diff
@@ -701,8 +710,13 @@ func setImplicitForVolume(vol *Volume, model Model) error {
 			vol.Structure[i].Offset = &start
 		}
 		previousEnd = *vol.Structure[i].Offset + quantity.Offset(vol.Structure[i].Size)
+		if quantity.Size(previousEnd) > minVolSize {
+			minVolSize = quantity.Size(previousEnd)
+		}
 	}
 
+	vol.MinSize = &minVolSize
+
```

Comment:
is there an explicit test for this computation? I'm not sure I see one

Reply:
I have added a test for the unordered partition case, but anyway testing is done already in all tests where MinSize is checked now, which are a few.